apps/demos/Demos/Charts/FunnelChart/description.md

```diff
@@ -1,6 +1,20 @@
-The DevExtreme JavaScript Funnel Chart visualizes a&nbsp;value at&nbsp;different stages. It&nbsp;helps assess value changes throughout these stages and identify potential issues. In&nbsp;this demo, the Funnel shows the percentage of&nbsp;website users that performed a&nbsp;certain action: downloaded a&nbsp;trial, contacted support, purchased a&nbsp;subscription, etc. The chart shows that most users decided not to&nbsp;subscribe after contacting support. You can use this information to&nbsp;make a&nbsp;decision, for example, to&nbsp;provide extra training for the support team.
+DevExtreme Funnel displays data in a funnel chart. You can implement funnel charts to display the flow of data over different stages. This demo displays conversion rates from website visits to product renewals.
```

Comment:
> DevExtreme Funnel displays data in a funnel chart.

Sounds like "X is so X". Plus, we can combine the first two sentences, for example: DevExtreme Funnel display the flow of data over different stages.